src/core/public/injected_metadata/injected_metadata_service.test.ts

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+import { DiscoveredPlugin } from 'src/core/server';
```

Comment:
Is this a TypeScript feature to allow imports from the root of the project like this?

Reply:
Hmm yeah I think my editor did that automatically, I'll make it relative.